### ISSUE TITLE
Reimplement problem.

### ISSUE THREAD

**Issue by tiexuedanxin** [GeeeG/CloudPose#5]
hello I have reimplement your network using pytorch, and use torch.nn.Conv1d instead tf.nn.conv2d, but the traning loss is much larger than your, could you give me some advice on how to solve the problem. Another question your paper say you using ICP to refine the result, but I can't find the ICP code in your project.

**Comment by GeeeG** [comment]
Hi,
1. Did you manage to get the training data from google drive?
2. I don't have any experience with pytorch so it's really hard to infer. Could you please give more info on your implemetation, and also the loss functions
3. The ICP is not in the git yet, I will update when I'm ready with the evaluation section, I use the open3d point to point icp for 10 iterations, you can check it out here http://www.open3d.org/docs/release/tutorial/Basic/icp_registration.html#Point-to-point-ICP

**Comment by gilad011** [comment]
Hi all, 
I've also try to implement my code on a custom dataset. 
@GeeeG am i able to create ftrecords files from my own dataset? how the files should be arranged?
@tiexuedanxin have you created a stable pytorch version that can help me out? could you share it?

**Comment by GeeeG** [comment]
Hi @gilad011 , yes, you can follow the "create your own training data" section. It contains the scripts to convert ycb video dataset (rgb, depth, label images etc.) into tfrecords, you can adjust accordingly to your dataset
